src/backend/Dockerfile

```diff
@@ -51,7 +55,10 @@ RUN set -ex \
 # Optional: Verify psql version during build to confirm success
 RUN psql --version
 
-ENTRYPOINT ["/usr/local/bin/gunicorn"]
-
+# Copy Django project source code
 COPY ./src/backend /usr/local/src/backend
-COPY ./src/frontend/build /usr/local/src/backend/static
+
+# Expose the port Gunicorn server will listen on
+EXPOSE 8085
```

Comment:
Do we need this too if we have [gunicorn.conf.py](https://github.com/azavea/echo-locator/blob/feature/asu/bootstrap-new-frontend/src/backend/gunicorn.conf.py)?

Reply:
Removed in https://github.com/azavea/echo-locator/pull/666/commits/0d9d7621d5399680ff38e2802c9ab729df8a02f6